components/sup/src/supervisor.rs

```diff
@@ -23,17 +23,17 @@ use std::fs::{self, File};
 use std::io::BufReader;
 use std::io::prelude::*;
 use std::path::PathBuf;
-use std::process::{Command, Stdio, Child};
+use std::process::Child;
 use std::thread;
 
 use hcore;
-use hcore::os::{process, users};
+use hcore::os::process;
 use hcore::package::PackageIdent;
 use libc::c_int;
 use time::{Duration, SteadyTime};
 
 use error::{Result, Error};
-use util::signals;
+use util::{create_command, signals};
```

Comment:
e.g. here, signals is a module that is being imported vs. create_command which is a function.